src/test/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -408,6 +408,232 @@ describe("Graph - Axes", () => {
             const rangeY2 = 250;
             expect(getAverageTicksCount(rangeY, rangeY2)).toEqual(5);
         });
+        it("uses custom tick values when axis.y.ticks.values is set", () => {
+            const localeAxisObj = utils.deepClone(axisDefault);
+            localeAxisObj.y = {
+                label: "y axis - custom ticks",
+                lowerLimit: 0,
+                upperLimit: 10,
+                ticks: {
+                    values: [2, 4, 6, 8]
+                }
+            };
+            graph = new Graph(
+                Object.assign(
+                    {
+                        ticksCount: 2
+                    },
+                    getAxes(localeAxisObj)
+                )
+            );
+            const allYAxisElements = document.querySelectorAll(
+                `.${styles.axisY}`
+            );
+            const NumberOfTicks = allYAxisElements[0].childNodes.length;
+            expect(
+                allYAxisElements[0].childNodes[1].querySelector("text")
+                    .textContent
+            ).toBe("2");
+            expect(
+                allYAxisElements[0].childNodes[NumberOfTicks - 1].querySelector(
+                    "text"
+                ).textContent
```

Comment:
We can modify the tests
```js
describe("when Y-2 axis is present", () => {
   describe("cutsom y and y-2 tick values are provided and ticksCount property is not provided", () => {
         it("should calculate y and y-2 axis tick values based on the custom y and y-2 values", () => {
            .........
         }

   describe("cutsom y and y-2 tick values are provided and ticksCount property is provided", () => {
         it("should calculate y and y-2 axis tick values based on the custom y and y-2 values", () => {
            .........
         }

   describe("cutsom y and y-2 tick values are not provided and ticksCount property is provided", () => {
         it("should calculate y and y-2 axis tick values based on the ticksCount property", () => {
            .........
         }

   describe("cutsom y and y-2 tick values are not provided and ticksCount property is not provided", () => {
         it("should calculate y and y-2 axis tick values based on use D3.js to automatically", () => {
            .........
         }
}

describe("when Y-2 axis is not present", () => {
   describe("cutsom y tick values are provided and ticksCount property is not provided", () => {
         it("should calculate y tick values based on the custom y values", () => {
            .........
         }

   describe("cutsom y values are provided and ticksCount property is provided", () => {
         it("should calculate y axis tick values based on the custom y values", () => {
            .........
         }

   describe("cutsom y values are not provided and ticksCount property is provided", () => {
         it("should calculate y axis tick values based on the ticksCount property", () => {
            .........
         }

   describe("cutsom y tick values are not provided and ticksCount property is not provided", () => {
         it("should calculate y axis tick values based on use D3.js to automatically", () => {
            .........
         }
}
```

Reply:
Updated tests here:
https://github.com/cerner/carbon-graphs/pull/249/commits/f8b9b62e502a9b67a5be44e8677ad6a0c6aae847